odoo/tools/xml_utils.py

```diff
@@ -55,6 +57,27 @@ def resolve(self, url, id, context):
             return self.resolve_string(attachment.raw, context)
 
 
+def _validate_xml(env, url, path, xmls):
+    # Get the XSD data
+    if path:
+        with file_open(path) as file:
```

Comment:
```suggestion
        with file_open(path, filter_ext=('.xml',)) as file:
```
(or other if needed)